spr/spr_test.go

```diff
@@ -344,7 +362,15 @@ func TestSPRBasicFlowFourCommits(t *testing.T) {
 }
 
 func TestSPRMergeCount(t *testing.T) {
-	s, gitmock, githubmock, _, output := makeTestObjects(t)
+	for _, sync := range []bool{true, false} {
```

Comment:
I think making the two calls is a little cleaner than the for loop in this case. 
move the t.Run into the helper function and just call the two:
`testSPRMergeCount(t, false)
testSPRMergeCount(t, true)`

Reply:
Good call out. Fixed